### ISSUE TITLE
sometimes GTK warning appears when try to optimize a file

### ISSUE THREAD

**Issue by Amiralgaby** [flozz/yoga-image-optimizer#41]
### Bug description
sometimes, a GTK warning is logged when I try to optimize a file

The warnings appears but seems not a problem to the program
#### What happened

```
$ yoga-image-optimizer ~/Images/Papiers\ peints/dark-unie.opti.jpg 

(yoga-image-optimizer:4984): Gtk-WARNING **: 15:36:19.402: Failed to set text from markup due to error parsing markup: Value of 'size' attribute on <span> tag on line 2 could not be parsed; should be an integer no more than 2147483647, or a string such as 'small', not '75%'

(yoga-image-optimizer:4984): Gtk-WARNING **: 15:36:19.404: Failed to set text from markup due to error parsing markup: Value of 'size' attribute on <span> tag on line 2 could not be parsed; should be an integer no more than 2147483647, or a string such as 'small', not '75%'

(yoga-image-optimizer:4984): Gtk-WARNING **: 15:36:19.405: Failed to set text from markup due to error parsing markup: Value of 'size' attribute on <span> tag on line 2 could not be parsed; should be an integer no more than 2147483647, or a string such as 'small', not '75%'

```

However the optimization works and produces the output :)

#### Step to reproduce

* [x] Use a file who generate the warning

Open a image who will generate the warning with yoga-image-optimizer and Optimize

by example with this file : [http://gabriel-theuws.fr/dark-unie.opti.jpg](http://gabriel-theuws.fr/dark-unie.opti.jpg)

### System information

* Operating system (with version): Ubuntu 20.04
* How did you installed the software?
    * [x] From PyPI (`pip3 install yoga-image-optimizer`

* YOGA Image Optimizer version: 1.2.0

**Comment by flozz** [comment]
This bug is caused by missing feature in older GTK versions... I will have to detect the GTK version to not use the unsupported markups with older GTK...

Related issue: https://github.com/flozz/yoga-image-optimizer/issues/33

NOTE: Flatpak package not affected

**Comment by flozz** [comment]
The relative font size were introduced in Pango 1.49.0 (about 1 year ago):

* https://gitlab.gnome.org/GNOME/pango/-/blob/main/NEWS#L166

So I will check the Pango version and do not specify the font size in the markup with older versions.

**Comment by flozz** [comment]
This is now fixed in the v1.2.2 :)
